### ISSUE TITLE
Cannot connect to other device using PeerID

### ISSUE THREAD

**Issue by uszasty** [Koenvh1/ets2-local-radio#46]
It worked only once, and not even on first try

What's happening: I can connect to the server both locally and from another device in local network, but cannot take control over it using PeerID, as nothing happens. I've tested it in many configurations between Vivaldi 1.11.880.3, Midori 0.5.11, whatever the latest Edge version is, Chrome for Android 60.0.3112.116 and official Android app.

Every time I connect from new browser, this happens in Error log.txt:
`System.Net.HttpListenerException (0x80004005): Określona nazwa sieciowa już jest niedostępna
   w System.Net.HttpResponseStream.Write(Byte[] buffer, Int32 offset, Int32 size)
   w ETS2_Local_Radio_server.SimpleHTTPServer.Process(HttpListenerContext context)
Not found: C:\Program Files (x86)\ETS2 Local Radio\web\stations\stations-custom.js
`
(Error says "given network name is already taken")
But this happens on connection, and not when I'm trying to use PeerID, so I dunno, might ne irrelevant.

**Comment by Koenvh1** [comment]
The peering functionality does not pass through the server, but uses WebRTC (I added it back in the day when it didn't have its own server, but still used ETS2 Telemetry Server. I still want to refactor it some day to use websockets instead of WebRTC, but that takes a lot of time I don't feel like spending on it at the moment).

Anyhow, WebRTC is *somewhat* experimental, and might not be supported in your version of Webkit yet. Could you test whether it works from one browser (e.g. Chrome) to another browser (e.g. Firefox)? If that works, then it might work on your phone through the Chrome browser. Alternatively, you can open the console (F12 in your browser). It should show something like PeerJS: Something. That's all the logging from the peering function. The connection to the TURN server seems to drop at times (Lost connection to ws://...), and while I did implement something to reconnect, it does not seem to reliably work for everyone (unfortunately I'm not quite sure what the issue is). Refreshing the page does work in those cases though.

If it doesn't work the first time, refresh the page. I'm still looking into why it sometimes doesn't work.

So, to summarise:
- Test whether it works with two desktop browsers (as mobile webkit might not support it on your phone yet)
- Try refreshing the page

**Comment by uszasty** [comment]
Hm, that might explain it. Looks like both midori and edge does not support WebRTC (which is weird, because edge apperently should... so you might rethink that notification that shows in unsupported browsers, because it actually suggest trying edge)

Well, thank you anyway. This wouldn't happen if I hadn't had problems with my main browser which is vivaldi, but I can't listen to shoutcast radios on it, so there's nothihg to do to fix this...
